client/homescreen/stats-overview/install-jetpack-cta.js

```diff
@@ -78,24 +82,34 @@ export const JetpackCTA = ( {
 
 export const InstallJetpackCTA = () => {
 	const { updateUserPreferences, ...userPrefs } = useUserPreferences();
-	const { jetpackInstallState, isBusy } = useSelect( ( select ) => {
-		const { getPluginInstallState, isPluginsRequesting } = select(
-			PLUGINS_STORE_NAME
-		);
-		const installState = getPluginInstallState( 'jetpack' );
-		const busyState =
-			isPluginsRequesting( 'getJetpackConnectUrl' ) ||
-			isPluginsRequesting( 'installPlugins' ) ||
-			isPluginsRequesting( 'activatePlugins' );
+	const { canUserInstallPlugins, jetpackInstallState, isBusy } = useSelect(
+		( select ) => {
+			const { getPluginInstallState, isPluginsRequesting } = select(
+				PLUGINS_STORE_NAME
+			);
+			const installState = getPluginInstallState( 'jetpack' );
+			const busyState =
+				isPluginsRequesting( 'getJetpackConnectUrl' ) ||
+				isPluginsRequesting( 'installPlugins' ) ||
+				isPluginsRequesting( 'activatePlugins' );
+			const { getCurrentUser } = select( USER_STORE_NAME );
+			const currentUser = getCurrentUser() || {};
 
-		return {
-			isBusy: busyState,
-			jetpackInstallState: installState,
-		};
-	} );
+			return {
+				isBusy: busyState,
+				jetpackInstallState: installState,
+				canUserInstallPlugins:
+					currentUser.allcaps && currentUser.allcaps.install_plugins,
```

Comment:
Just a note that if you implement my other change with REST controller, this will become `currentUser.capabilities` instead of `currentUser.allcaps`.

Reply:
ok, thank you!